modules/flight_interface/flight_interface.py

```diff
@@ -21,27 +24,44 @@ def create(cls, address: str, timeout_home: float) -> "tuple[bool, FlightInterfa
         address: TCP address or port.
         timeout_home: Timeout for home location in seconds.
         """
+        result, flight_interface_logger = logger.Logger.create("flight_interface")
+        if not result:
+            return False, None
```

Comment:
I think the idea is we don't want to return False, None if the log creation fails, since it's not a critical part of the system, we don't want it taking down the system